TombEngine/Objects/TR3/Entity/FishSwarm.cpp

```diff
@@ -156,7 +156,7 @@ namespace TEN::Entities::Creatures::TR3
 					if (dist < closestDist &&
 						targetItem.ObjectNumber == ID_CORPSE &&
 						targetItem.Active && TriggerActive(&targetItem) &&
-						targetItem.ItemFlags[1] == (int)CorpseFlag::Grounded &&
+						targetItem.ItemFlags[7] == (int)CorpseFlag::Grounded &&
```

Comment:
I have noticed that Compsognathus object also uses `ItemFlags[1]` for setting `CorpseFlag`. Is that also a bug?

Reply:
yes, I completely forgot about the compies